CMakeLists.txt

```diff
@@ -41,7 +41,8 @@ macro(FIND_CURL)
       set (OLD_SUFFIXES ${CMAKE_FIND_LIBRARY_SUFFIXES})
       set (CMAKE_FIND_LIBRARY_SUFFIXES .a)
       find_package(CURL REQUIRED)
-      list(APPEND CURL_LIBRARIES ssl crypto)
+      find_package(OpenSSL REQUIRED)
```

Comment:
This will find `.a` only. Is it intended?

Reply:
Not intended, thanks.